static/content.json

```diff
@@ -50,153 +45,167 @@
+                    "about": "Ole Eskild har erfaring med en rekke teknologier og metodikker og har jobbet med alt fra API-tjenester og sky-infrastuktur til komplekse frontend applikasjoner. Han trives best med eierskap til hele teknologi-stacken, men liker også å bryne seg på rene backend eller frontend prosjekter. "
                 },
                 {
                     "imagePath": "/images/people/asbjorn.png",
                     "name": "Asbjørn Rekve",
-                    "title": "Consultant",
+                    "title": "Konsulent",
                     "email": "asbjorn@savvy.no",
-                    "about": "Asbjørn is a versatile developer with broad experience in the development of web solutions. Through projects at Sbanken, the Norwegian Armed Forces, Equinor, and previously as CTO for Powerzeek, he has faced challenging tasks and is used to a great deal of responsibility. He has experience of working both independently and in collaboration with others."
+                    "about": "Asbjørn er en allsidig utvikler med bred kompetanse innen utvikling av webløsninger. Gjennom prosjekter hos Guilty, Sbanken, Forsvaret, Equinor, og tidligere som CTO for Powerzeek, har han hatt utfordrende arbeidsoppgaver og er vant med stort ansvar. Han har erfaring med å jobbe både selvstendig og sammen med andre, og har spesielt god erfaring fra autonome team."
+                },
+                {
+                    "imagePath": "/images/header-image-1.png",
+                    "name": "Ny ansatt",
+                    "title": "Konsulent",
+                    "email": "",
+                    "about": "Vi får en ny ansatt i januar!"
                 },
                 {
                     "imagePath": "/images/header-image-1.png",
-                    "name": "New employee",
-                    "title": "Consultant",
+                    "name": "Ny ansatt",
+                    "title": "Konsulent",
                     "email": "",
-                    "about": "We have a new employee starting in February of 2023!"
+                    "about": "Vi får en ny ansatt i februar!"
                 }
             ]
         },
         "contact": {
-            "metaTitle": "Savvy | Contact us",
-            "metaDescription": "Contact us now please",
-            "text": "We are always looking for new opportunities. Do not hesitate to contact us if you are in need of a Savvy consultant, or want to work with us in any other way."
+            "metaTitle": "Savvy | Kontakt oss",
+            "metaDescription": "Kontakt oss",
+            "text": "Vi leter alltid etter nye muligheter. Ikke nøl med å kontakte oss dersom du har behov for en Savvy konsulent, eller ønsker å samarbeide med oss på andre måter."
```

Comment:
"Vi er alltid åpen for nye muligheter" ? Eller er det for passivt?

Reply:
Forsøksvis fikset.